build-release.sh

```diff
@@ -39,6 +39,7 @@ build_dist() {
 	# Clean out any old remnants of failed builds.
 	rm -rf "dist/$DIR"
 	mkdir -p "dist/$DIR"
+	trap "rm -rf \"dist/$DIR\"" INT TERM
```

Comment:
Not sure this quote escaping is good... shellcheck also complains

Reply:
We want to expand now, we don't know the value of "$DIR" at the time we get signal.

Reply:
Actually, a `trap` here seems like the wrong approach. Since this would only clean up issues from the current run. And cleaning issues from your build isn't ideal, you might want to leave the artifacts as they were, to investigate any failure...

Instead, I think we should simply not pass folders to the `CHECKSUM_CMD` command below, like this: https://stackoverflow.com/questions/20745656/globbing-for-only-files-in-bash

Reply:
Fixed.